doc/Quests.md

```diff
@@ -0,0 +1,43 @@
+# Quests (Planing Phase)
+
+To be able to interact and group up with other AIs on the server, there should
+be a way to gain reputation to define the level of cooperation.
+
+To gain reputation quests can be solved, which increase the reputation on
+successful solving these or decrease on failing.
+
+A `Quester` creep attaches the Quest start at controllers in vacant rooms, like:
+
+    {"type": "Quest", "id": 0.3451, "origin": "W1N7", info: "http://tooangel.github.io/screeps/doc/Quests.html"}
+
+To apply for the Quest, remove the message from the controller and send a message
+via Terminal transfer to the `origin` room.
+
+   {"type": "Quest", "id": 0.3451, "room": "W2N7"}
```

Comment:
Actually no, the mentioned room should be the one where you found the quest, so that we can check if there is a quest attached, to prevent just sending these kinds of messages and we loose control over the quests
Will improve the documentation.